### ISSUE TITLE
Scope Reverting to "project" on Save

### ISSUE THREAD

**Issue by patorpey** [mikaeljorhult/brackets-todo#128]
In my preferences for brackets-todo (and confirmed to be persistent in `brackets.json`), I have selected "Current file" for Search Scope. This works as expected when switching files, as I see only the TODO comments for that file. However, after editing a file and saving the file, brackets-todo updates listing all TODO comments for the entire project instead of still only those for the current file. Switching files or opening and accepting the settings dialog for brackets-todo again displays only the items for the current file, as desired.

**Comment by mikaeljorhult** [comment]
Are you still experiencing this in the latest version of Brackets and the extension?

**Comment by patorpey** [comment]
Yes, I am still experiencing this in Brackets 1.10.0-17348 and Todo v0.9.3.

To clarify, the view does not change to the "Current project" view with files listed that can be twirled down to reveal their TODO items. However, the normal list view, after saving a change that affects TODOs in the active file, will list individual TODO items for all files in the project, displayed as if they were in the current file.
